gtsfm/utils/ellipsoid.py

```diff
@@ -117,17 +117,15 @@ def get_alignment_rotation_matrix_from_svd(point_cloud: np.ndarray) -> np.ndarra
     return wuprightRw
 
 
-def get_right_singular_vectors(A: np.ndarray) -> np.ndarray:
-    """Extracts the right singular eigenvectors from the point cloud. Some of the eigenvectors could be randomly
-    multiplied by -1. Despite this, the eigenvectors will still remain valid.
-
-    Ref: https://stackoverflow.com/questions/18152052/matlab-eig-returns-inverted-signs-sometimes
+def sorted_eigendecomposition(A: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
+    """Extracts the eigendecomposition results sorted by eigenvalues in descending order
 
     Args:
         A: point cloud of shape (N,3)
 
     Returns:
-        The right singular vectors of the point cloud, shape (3,3).
+        eigvals: The sorted eigenvalues of the point cloud, shape (3,3).
+        eigvecs: The sorted eigenvectors of the point cloud, shape (3,3).
```

Comment:
the eigenvectors of A^T*A are the singular vectors, so the name was actually correct previously